### PR TITLE
remove default geometry in ogr source

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[OgrSource: Default-Geometry rauswerfen](https://trello.com/c/n70R8B6c/145-ogrsource-default-geometry-rauswerfen)

File: operators/src/source/ogr_source.rs [hunk 1]
```diff
@@ -752,9 +744,9 @@ where
                         Ok(Some(FieldValue::Integer64Value(v))) => Some(v.to_string()),
                         Ok(Some(FieldValue::StringValue(s))) => Some(s),
                         Ok(Some(FieldValue::RealValue(v))) => Some(v.to_string()),
-                        Ok(Some(_)) => return Err(Error::OgrColumnFieldTypeMismatch), // TODO: handle other types
                         Ok(None) => None,
-                        Err(_) => None, // TODO: log error
+                        Ok(Some(_)) => error_spec.on_error(Error::OgrColumnFieldTypeMismatch)?,
+                        Err(e) => error_spec.on_error(Error::Gdal { source: e })?, // TODO: handle other types
```
Comment (ChristianBeilschmidt) [inline operators/src/source/ogr_source.rs:749]: move todo back to `Ok(Some(_)) => …`

File: operators/src/source/ogr_source.rs [hunk 2]
```diff
@@ -765,9 +757,9 @@ where
                         Ok(Some(FieldValue::IntegerValue(v))) => Some(f64::from(v)),
                         Ok(Some(FieldValue::StringValue(s))) => f64::from_str(&s).ok(),
                         Ok(Some(FieldValue::RealValue(v))) => Some(v),
-                        Ok(Some(_)) => return Err(Error::OgrColumnFieldTypeMismatch), // TODO: handle other types
                         Ok(None) => None,
-                        Err(_) => None, // TODO: log error
+                        Ok(Some(_)) => error_spec.on_error(Error::OgrColumnFieldTypeMismatch)?,
+                        Err(e) => error_spec.on_error(Error::Gdal { source: e })?, // TODO: handle other types
```
Comment (ChristianBeilschmidt) [inline operators/src/source/ogr_source.rs:762]: move todo back to `Ok(Some(_)) => …`

File: operators/src/source/ogr_source.rs [hunk 3]
```diff
@@ -779,9 +771,9 @@ where
                         Ok(Some(FieldValue::Integer64Value(v))) => Some(v),
                         Ok(Some(FieldValue::StringValue(s))) => i64::from_str(&s).ok(),
                         Ok(Some(FieldValue::RealValue(v))) => Some(v as i64),
-                        Ok(Some(_)) => return Err(Error::OgrColumnFieldTypeMismatch), // TODO: handle other types
                         Ok(None) => None,
-                        Err(_) => None, // TODO: log error
+                        Ok(Some(_)) => error_spec.on_error(Error::OgrColumnFieldTypeMismatch)?,
+                        Err(e) => error_spec.on_error(Error::Gdal { source: e })?, // TODO: handle other types
```
Comment (ChristianBeilschmidt) [inline operators/src/source/ogr_source.rs:776]: move todo back to `Ok(Some(_)) => …`

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]
you just misplaced some todos, but the rest is fine

File: operators/src/source/ogr_source.rs (hunk 1 above)
Comment (michaelmattig) [reply operators/src/source/ogr_source.rs]: 8f95de873564abf89e0b7752302b1ab30ebb97fa

File: operators/src/source/ogr_source.rs (hunk 2 above)
Comment (michaelmattig) [reply operators/src/source/ogr_source.rs]: 8f95de873564abf89e0b7752302b1ab30ebb97fa

File: operators/src/source/ogr_source.rs (hunk 3 above)
Comment (michaelmattig) [reply operators/src/source/ogr_source.rs]: 8f95de873564abf89e0b7752302b1ab30ebb97fa

**Pull request merged**